.github/actions/test-fc/action.yml

```diff
@@ -39,7 +39,23 @@ runs:
             args="-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib"
           fi
         fi
+        
+        ${{ env.FC }} $args -o hw hw.f90
+        output=$(./hw '2>&1')
+        [[ "$output" == *"hello world"* ]] && echo "$output" || (echo "Unexpected Fortran program output: $output"; exit 1)
+        rm hw
+
+    - name: Test compile (bash) (intentionally failing)
```

Comment:
git is displaying the diff a bit strangely - I've added this one run command here which moves the library back to its original place to demonstrate the failure. This is to be removed ahead of merging the changes in hw.f90 and setup_fortran.sh